person.rb

```diff
@@ -0,0 +1,23 @@
+class Person 
+  attr_reader :id, :name, :age
```

Comment:
- [ ] :name and :age are already defined on attr_accessor which is both attr_reader and attr_getter so please remove name and age from attr_reader